### DISCUSSION TITLE
People Search using People entity from Microsoft Search data source

### DISCUSSION THREAD

**Discussion by anujabhojani** [microsoft-search/pnp-modern-search#3824] [Q&A]
Hi Team,

I am trying to use the People entity from Microsoft Search data source to configure People Search for our organization.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/3500375/84425769-5b00-4416-93c3-3c9931fe3b7d)
However, getting 0 results with default *, works when specific user name provided.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/3500375/72b12907-cb78-46ae-a3c4-5b2f5d0ce020)

All required Graph API permission have been given:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/3500375/1c056f60-0be6-439f-abc0-1790c3761980)
 

Can someone please let me know how to use the people entity, and required changes for the layout?

**Comment by kasperbolarsen** [comment]
The wildcard for Microsoft people search is a ?, not the usual *

Secondly, I will recommend using SharePoint as the datasource as MS search is a pain in the behind for people search
